spec/unit/actions/synchronize_users_spec.rb

```diff
@@ -130,5 +130,22 @@
         let(:message_for_project) { :import_users }
       end
     end
+    context 'when using mistyped mode' do
+      let(:params) do
```

Comment:
Similar code found in synchronize_users_spec.rb:76, synchronize_users_spec.rb:134 (mass = 40)